### ISSUE TITLE
[A11y]_.Json_Version History_Reflow: On applying reflow, Version history tooltip in "Version" tab is not visible.

### ISSUE THREAD

**Issue by lyndaidaii** [NuGet/NuGetGallery#9449]
Details: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1744522

**Comment by lyndaidaii** [comment]
duplicate as [#9424](https://github.com/NuGet/NuGetGallery/issues/9424)
